package.json

```diff
@@ -39,6 +39,13 @@
       "default": false,
       "description": "Remember the typed query when closing the fuzzy finder and use that as the starting query next time the fuzzy finder is opened."
     },
+    "maxItem": {
```

Comment:
Can this be renamed to `maxItems` instead?  That way when it's displayed it'll read as "Max Items" and not "Max Item".

In addition, perhaps items can be replaced with "files"?  So "maxFiles" and "Maximum number of files to be displayed"?

Just some :thought_balloon:s.